### PR TITLE
Templates: Added static Entertainment template

### LINKED ISSUE #1477: Templates: Entertainment
Create the Entertainment template from
https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=1%3A5

image

### PR DESCRIPTION
**Pull request by mariano-formidable**
This PR adds the static Entertainment template.

Fixes #1477 

image

### CODE REVIEW

**Review by BrittanyIRL — COMMENTED** [review]
This template looks like it was really complicated! I have a few nits, but feel free to ignore them if they're not possible within the tools we have currently - if that's the case I'll go ahead and approve this. It looks really great, if not for the design attached I would never have seen anything off with it.

1. On that first page it looks like the white text is supposed to butt up exactly to the edge of the story, in the dashboard it looks like there's a little bit of space? Is this possible to fix? 
**Dashboard:** 
Screen Shot 2020-05-05 at 8 43 57 AM
**Design:** 
Screen Shot 2020-05-05 at 8 45 09 AM

2. The quote on page 3 seems like the font is wrong
**Dashboard**
Screen Shot 2020-05-05 at 8 46 12 AM
**Design**
Screen Shot 2020-05-05 at 8 46 19 AM

3. Page 5 looks like the image is cropped a little at the top. (same for the image on page 6) 
**Dashboard**
Screen Shot 2020-05-05 at 8 49 33 AM
**Design**
Screen Shot 2020-05-05 at 8 49 37 AM

4. The pink borders are real thick on page 8
**Dashboard**
Screen Shot 2020-05-05 at 8 51 52 AM
**Design**
Screen Shot 2020-05-05 at 8 51 56 AM

5. The last page is missing the blue border on the middle item.
**Dashboard**
Screen Shot 2020-05-05 at 8 53 05 AM
**Design**
Screen Shot 2020-05-05 at 8 53 09 AM

**Comment by mariano-formidable** [comment]
Good nits, I can fix most of them, but some of them aren't fixable.

1.) Getting the alignment right to the edge is tricky, but I should be able to squeeze it right up to the edge better.  I'll fix!

2.) Oddly enough, the quote in the design and the quote in my version are the _same_ character and supposedly in the _same_ font!!!  I think the design took liberties with the font that we can't do in the editor... So, can't fix :/

3.) I aligned the image to the super edge, instead of the inner edge.  I'll fix!

4.)  Wooooah, I'm not sure what happened to that page.  I must have adjusted something and things went out of whack and I didn't notice.  I can get the pink bars to display correctly.  I'll fix!

5.) Hmmmm... I think that blue bar might be an "onhover" thing (which the editor doesn't support), but I'm curious where you pulled this image from because the figma guildlines don't have any blue borders on the last page:
image

So, I'll leave it as is.

Here's the source of "truth" we're building off of:
https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=1%3A5

**Comment by BrittanyIRL** [comment]
> Here's the source of "truth" we're building off of:
> https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=1%3A5

Ooops! I was referencing the screenshot here on the ticket referenced in your description: https://github.com/google/web-stories-wp/issues/1477

**Comment by mariano-formidable** [comment]
> > Here's the source of "truth" we're building off of:
> > https://www.figma.com/file/RTzSw47oi6pEZXEfP2JoBo/Templates?node-id=1%3A5
> 
> Ooops! I was referencing the screenshot here on the ticket referenced in your description: #1477

Ha, looks like that's a different page in figma.  I'll update the original ticket with a screenshot from where I'm working from :)

File: assets/src/dashboard/templates/data/fitness.js [hunk 1]
```diff
@@ -740,14 +740,13 @@ export default function (imageBaseUrl) {
...
-                'http://localhost:8899/wp-content/uploads/2020/04/fashion_page3_arrow.png',
+              src: `${imageBaseUrl}assets/images/templates/fitness/fitness_page3_arrow.png`,
               width: 7,
               height: 3,
               posterId: 0,
               id: 64,
-              title: 'fashion_page3_arrow',
-              alt: 'fashion_page3_arrow',
+              title: 'fitness_page3_arrow',
+              alt: 'fitness_page3_arrow',
```
Comment (carloskelly13) [inline assets/src/dashboard/templates/data/fitness.js:749]: Good find

**Review by carloskelly13 — CHANGES_REQUESTED** [review]
Just a couple text issues

1. The repeated "Out Now" should have the top portion duplicated more as shown in the second picture

Screen Shot 2020-05-05 at 12 22 56 PM

Screen Shot 2020-05-05 at 12 23 33 PM


2. The Ts seem misaligned here

Screen Shot 2020-05-05 at 12 22 35 PM

**Comment by mariano-formidable** [comment]
> Just a couple text issues
> 
> 1. The repeated "Out Now" should have the top portion duplicated more as shown in the second picture
> 
> Screen Shot 2020-05-05 at 12 22 56 PM
> 
> Screen Shot 2020-05-05 at 12 23 33 PM
> 
> 1. The Ts seem misaligned here
> 
> Screen Shot 2020-05-05 at 12 22 35 PM

1.) Yeah, the repeated text effect isn't something we can replicate in the editor as it is in the design.  I've noted it in the "missing features" document we have to note things we can't do in our templates.  Can't fix :/

2.) I fixed the alignment issue, good catch!  Fixed!

**Review by carloskelly13 — APPROVED** [review]

**Review by BrittanyIRL — APPROVED** [review]
Looks so good!! thanks for making those changes!

**Pull request merged**